### PR TITLE
[zh-tw] Localize 'Cloud Computing'

### PR DESCRIPTION
**Pull request by rosawu159**
### Describe your changes

Add the zh-tw localization for the cloud-computing page,
Original comes from https://glossary.cncf.io/cloud-computing/

### Related issue number or link

#2054 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/cloud-computing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 雲端運算
+status: Completed
+category: 概念
+tags: ["方法論", "基礎設施", ""]
+---
+
+## 什麼是雲端運算 {#what-it-is}
+
+雲端運算是一種通過網際網路提供計算資源（如CPU、網路和磁碟功能）的服務，使用戶可以遠端存取位於實體位置的運算資源。一般會根據雲端基礎設施是用於專門為組織而設或是共享用於公開服務，將其區分為私有雲和公有雲。
```
Comment (pichuang) [inline content/zh-tw/cloud-computing.md:10]:
```suggestion
雲端運算是一種通過網際網路提供計算資源（如CPU、網路和磁碟功能）的服務，使用戶可以遠端存取位於實體位置的運算資源。一般會根據雲端基礎設施是用於專門為組織而設或是共享用於開放式服務，將其區分為私有雲和公有雲。
```

File: content/zh-tw/cloud-computing.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+普遍來說，當組織需要擴展其運算資源時，會面臨兩個主要挑戰：一方面是購買、支援和設計新的設施來托管物理伺服器和網絡，另一方面則是擴充和維護現有設施。透過雲端運算，組織能夠透過外包部分的運算需求以解決這些考驗。
```
Comment (pichuang) [inline content/zh-tw/cloud-computing.md:14]:
```suggestion
普遍來說，當組織需要擴展其運算資源時，會面臨兩個主要挑戰：一方面是購買、支援和設計新的設施來托管物理伺服器和網路，另一方面則是擴充和維護現有設施。透過雲端運算，組織能夠透過外包部分的運算需求以解決這些考驗。
```

**Review by pichuang — APPROVED** [review]
Hi @rosawu159 would you mind rebasing 5 commits into 1 commit?

**Comment by pichuang** [comment]
Hi @rosawu159 Thank you for your effort and PR. Would you mind clearing your git comment because there is quite a lot of repetitive content in one commit.

image

**Comment by pichuang** [comment]
@ydFu would you mind take a look?

**Review by hwchiu — APPROVED** [review]

**Review by ydFu — APPROVED** [review]
LGTM

**Pull request merged**